### PR TITLE
[plugin] Add WebexTeams alert plugin documentation

### PR DESCRIPTION
**Pull request by liqingwang**
apache/dolphinscheduler#8215

https://github.com/apache/dolphinscheduler/pull/8217

### CODE REVIEW

**Review by dailidong — APPROVED** [review]
+1
good job

File: docs/en-us/dev/user_doc/guide/alert/enterprise-webexteams.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# WebexTeams
+
+If you need to use WebexTeams to alert, please create an alarm Instance in warning instance manage, and then choose the WebexTeams plugin. The configuration example of enterprise WebexTeams is as follows:
+
+![enterprise-wechat-plugin](/img/alert/enterprise-wechat-plugin.png)
```
Comment (zhongjiajie) [inline docs/en-us/dev/user_doc/guide/alert/enterprise-webexteams.md:5]: Seem it is a wrong link and link text
```suggestion
![enterprise-webexteams-plugin](/img/alert/enterprise-webexteams-plugin.png)
```

**Review by zhongjiajie — DISMISSED** [review]

File: docs/en-us/dev/user_doc/guide/alert/enterprise-webexteams.md (hunk 1 above)
Comment (zhongjiajie) [reply docs/en-us/dev/user_doc/guide/alert/enterprise-webexteams.md:5]: Hi @liqingwang, could you change to webexteams img?

Comment (liqingwang) [reply docs/en-us/dev/user_doc/guide/alert/enterprise-webexteams.md]: Hi @zhongjiajie I am sorry, it was a mistake, I have corrected it now.

**Review by zhongjiajie — APPROVED** [review]
LGTM, thanks @liqingwang

**Pull request merged**